test/tests/ver-1fc/ver-1fc.i

```diff
@@ -0,0 +1,166 @@
+T_SA = 600 # K
+T_SB = 0 # K
+L_A = 40e-2 # m
+L_B = 40e-2 # m
+k_A = 401 # W/m/K
+k_B = 80.2 # W/m/K
+num_nodes = 800 # (-)
+position_measurement = 9e-2 # m
+
+[Mesh]
+  [whole_domain]
+    type = GeneratedMeshGenerator
+    xmin = 0
+    xmax = ${fparse L_A + L_B}
+    dim = 1
+    nx = ${num_nodes}
+  []
+  [injection_block_1]
+    type = ParsedSubdomainMeshGenerator
+    input = whole_domain
+    combinatorial_geometry = 'x < ${L_A}'
+    block_id = 0
+  []
+  [injection_block_2]
+    type = ParsedSubdomainMeshGenerator
+    input = injection_block_1
+    combinatorial_geometry = 'x > ${L_A}'
+    block_id = 1
+  []
+[]
+
+[Variables]
+  [temperature]
+    initial_condition = 0.0
+    scaling = 1e-6
+  []
+[]
+
+[Functions]
+  [thermal_conductivity_func_Cu]
+    type = ParsedFunction
+    expression = ${k_A}
+  []
+  [thermal_conductivity_func_Fe]
+    type = ParsedFunction
+    expression = ${k_B}
+  []
+[]
+
+[Kernels]
+  [heat]
+    type = HeatConduction
+    variable = temperature
+  []
+  [HeatTdot]
+    type = HeatConductionTimeDerivative
+    variable = temperature
+  []
+[]
+
+[BCs]
+  [lefttemperature]
+    type = DirichletBC
+    boundary = left
+    variable = temperature
+    value = ${T_SA}
+  []
+  [righttemperature]
+    type = DirichletBC
+    boundary = right
+    variable = temperature
+    value = ${T_SB}
+  []
+[]
+
+[Materials]
+  [specific_heat_Cu]
+    type = GenericConstantMaterial
+    block = '0'
+    prop_names = 'density specific_heat'
+    prop_values = '8940.0 384.70'
+  []
+  [specific_heat_Fe]
+    type = GenericConstantMaterial
+    block = '1'
+    prop_names = 'density specific_heat'
+    prop_values = '7860.0 447.57'
```

Comment:
Please mention units here.

Reply:
I moved all the values to the top of the input file, and added the units. Good catch!